app/controllers/provider_interface/reports_controller.rb

```diff
@@ -1,5 +1,7 @@
 module ProviderInterface
   class ReportsController < ProviderInterfaceController
-    def index; end
+    def index
+      @providers = current_user.providers
```

Comment:
Can anyone access this export for their organsiation(s) regardless of permissions?

Reply:
It will not let you access the page, but it will produce an Activerecord error because it cant find an ID associated with you. this will need fixing

Reply:
Sorry that might not have been a clear question:
Are specific permissions necessary to access this page? I don't see a check for any specific permissions, though this may not be necessary.

Reply:
Currently there is no check, does there need to be a specific permission for this page? -> Its not personally identifiable data so wouldnt fall under view diversity or view hesa data, seems extreme to require manage users for tallying application numbers

Reply:
Ah yes it seems other exports/reports don't check for additional permissions so that's all good.